.gitignore

```diff
@@ -22,9 +23,9 @@ tags
 /docs/build
 /ployst.egg-info
 /static
+/.tox/
 
 # Some OSX specific thing
 *.DS_Store
-/.tox/
 
 ployst/github/test/data/
```

Comment:
I'm not sure why we ignore this. There are some files here under version control.
Do you know @alexcouper ?